gnucash/import-export/bi-import/dialog-bi-import.c

```diff
@@ -231,7 +231,6 @@ gnc_bi_import_fix_bis (GtkListStore * store, guint * fixed, guint * deleted,
     gboolean valid, row_deleted, row_fixed;
     gchar *id = NULL, *date_opened = NULL, *date_posted = NULL, *due_date = NULL,
         *owner_id = NULL, *date = NULL, *quantity = NULL, *price = NULL;
-    GString *prev_id, *prev_date_opened, *prev_date_posted, *prev_owner_id, *prev_date;	// needed to fix multi line invoices
```

Comment:
This intermediary commit got me on the wrong track... The final new behaviour of the row validation function is actually defined by the next commit.
Please squash these two commits together so we only see the final and intended behaviour. This intermediate step is confusing to a reviewer.

Reply:
Commits have been squashed together, thanks.